com/truckmuncher/api/trucks.proto

```diff
@@ -77,6 +85,19 @@ message ActiveTrucksResponse {
          * Value in the range [-180, 180]
          */
         required double longitude = 3;
+        /** 
+         * Indicates whether the truck's location is verified or not.
+         */
+        optional bool verified = 4;
+        /**
+         * Indicates the current upvote/downvote score.
```

Comment:
No, negatives shouldn't be returned; I was thinking 0s could be, but I guess that's up for debate.